clang/lib/StaticAnalyzer/Checkers/WebKit/RawPtrRefMemberChecker.cpp

```diff
@@ -146,13 +149,67 @@ class NoUncountedMemberChecker
     BR->emitReport(std::move(Report));
```

Comment:
`setDeclWithIssue()` goes into a different PR right?

Reply:
I'm not calling `setDeclWithIssue` in this checker since the checker works on classes, and not functions. I guess there could be classes declared within a function?